app/upgrades/v16/proposals.go

```diff
@@ -9,12 +9,19 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	govkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
-	incentives "github.com/evmos/evmos/v16/x/incentives/types"
 )
 
-// DeleteIncentivesProposals deletes the RegisterIncentives & CancelIncentiveProposal proposals from the store
-// because the module was deprecated
-func DeleteIncentivesProposals(ctx sdk.Context, gk govkeeper.Keeper, logger log.Logger) {
+// deprecatedProposals is a map of the TypeURL
+// of the deprecated proposal types
+var deprecatedProposals = map[string]struct{}{
+	"/evmos.incentives.v1.RegisterIncentiveProposal": {},
+	"/evmos.incentives.v1.CancelIncentiveProposal":   {},
+	"/evmos.erc20.v1.RegisterCoinProposal":           {},
+}
+
+// DeleteDeprecatedProposals deletes the RegisterCoin, RegisterIncentives & CancelIncentiveProposal
+// proposals from the store because were deprecated
+func DeleteDeprecatedProposals(ctx sdk.Context, gk govkeeper.Keeper, logger log.Logger) {
 	// Delete the only incentives module proposals
```

Comment:
nit: outdated and redundant imo
```suggestion
```

Reply:
Also this one is outdated:
https://github.com/evmos/evmos/blob/6ca5c913f4ca6e19565352d19fbab7a75b9e4533/app/upgrades/v16/proposals.go#L27-L27